client/js/array-util.js

```diff
@@ -1,12 +1,24 @@
+//Returns array from fromNum to toNum that is sequential in order, both increasing and decreasing
 const getRange = function(fromNum, toNum) {
```

Comment:
Excellent. For these files, you'll also need to adjust them (since they have already been created) with the bottom right button that lets you select "Tab Width: 2" and then "Convert indentation to spaces".